docs/source/what_is_jupyter.md

```diff
@@ -0,0 +1,412 @@
+- Testing newly released versions
+- Adding software features
+- Hosting the weekly video meetings (LINK)
+- Helping others in the community
+- ...and more!
+
+The project is split into largely independent subprojects which handle
+different aspects of Jupyter software and the community. A central council,
+the Jupyter Executive Council (LINK), makes decisions about project-wide
+goals and policies, while different subprojects handle the actual development
+of the various software components.
+
+Some subprojects take care of broader topics, such as the [Accessibility](),
+[Security](), [Community](), and [Documentation]() projects.
+
+# A (Partial) Tour of the Jupyter Universe
+
+In the following sections, we are going to look at some popular components of
+the Jupyter ecosystem. This is *not* a comprehensive reference of every aspect
+of Jupyter, but rather a big-picture summary that should help illustrate some
+important parts that were discussed earlier.
+
+If you are completely new to *Jupyter*, the project's [About][] page is good
+reading that will introduce you to many of the different subprojects.
+
+If you want to know more about the organizational structure, check out the
+[Governance]() pages.
+
+And if you would like to contribute to the project, have a look at the
+[Get Involved]() page.
+
+The graph below presents the best known software components of the Project.
+
+Connections and groups in this diagram are not formal relationships, but
+simple indicators to help us draw the big picture in the next sections.
+
+```mermaid
+graph TD
+    IPython
+        click IPython href "https://ipython.org" _blank
+
+    subgraph source/code
+        Nbformat[["Notebook file format (.ipynb)"]]
+            click Nbformat href "https://nbformat.readthedocs.io" _blank
+
+        JupyterWidgets>Jupyter-Widgets]
+            click JupyterWidgets href "http://ipywidgets.readthedocs.io" _blank
+    end
+
+    subgraph frontends
+        direction LR
+
+        JupyterLab(Jupyter-Lab)
+            click JupyterLab href "https://jupyterlab.readthedocs.io" _blank
+
+        JupyterNotebook(Jupyter-Notebook)
+            click JupyterNotebook href "https://jupyter-notebook.readthedocs.io" _blank
+    end
+
+    subgraph viewers
+        direction LR
+
+        Binder(Binder)
+            click Binder href "https://mybinder.org" _blank
+
+        Voila(Voilà)
+            click Voila href "https://voila.readthedocs.io" _blank
+
+        Nbviewer[nbviewer]
+            click Nbviewer href "https://nbviewer.org" _blank
+
+        Nbconvert[nbconvert]
+            click Nbconvert href "https://nbconvert.readthedocs.io" _blank
+    end
+
+    JupyterHub(Jupyter-Hub)
+        click JupyterHub href "https://jupyterhub.readthedocs.io" _blank
+
+    JupyterHub --> frontends
+    IPython ~~~ source/code
+    frontends --> source/code
+    viewers --> source/code
+```
+
+### Notebook Editor Programs
+
+- [JupyterLab](https://jupyterlab.readthedocs.io/en/latest/)
```

Comment:
Is it too much to characterize JupyterLab as an IDE in this section?

Reply:
Going by the target audience again (maybe a chemist who does a little python for their lab work), I think this is unnecessary jargon that would hurt communication: it introduces a new idea, a complex one, and throws uncertainty onto the nature of JupyterLab, where I think editor is a plain english alternative that works better here.